.github/PR_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+### Proposed change(s)
+
+Describe the changes made in this PR. If this is a bug fix or a new feature please provide a link
```

Comment:
"provide a link to it" - link to what?

Maybe add a link section with items for github issues, forum threads, and jira tickets.

Reply:
added a section.